pkg/frequencyplans/frequencyplans.go

```diff
@@ -198,16 +199,20 @@ func (lsc *LoRaStandardChannel) Clone() *LoRaStandardChannel {
 }
 
 // ToConcentratorConfig returns the LoRa standard channel configuration in the protobuf format.
-func (lsc *LoRaStandardChannel) ToConcentratorConfig(band band.Band) *ttnpb.ConcentratorConfig_LoRaStandardChannel {
+func (lsc *LoRaStandardChannel) ToConcentratorConfig(phy band.Band) *ttnpb.ConcentratorConfig_LoRaStandardChannel {
 	if lsc == nil {
 		return nil
 	}
-	dr := band.DataRates[lsc.DataRate].Rate.GetLoRa()
+	dr, ok := phy.DataRates[ttnpb.DataRateIndex(lsc.DataRate)]
+	if !ok {
+		panic(fmt.Sprintf("data rate '%d' does not exist", lsc.DataRate))
```

Comment:
Prefer returning err instead of panicking

Reply:
The original code simply used the data rate directly without checking anything, so I assumed that this is expected to always exist and panicking is arguably better than silently using undefined value.
I will try to adapt this to use errors.